### PR TITLE
Fix leftover from #13

### PR DESCRIPTION
**Pull request by aabadie**
There were remaining unaddressed comments in #13. This PR tries to fix them plus other things:
* in README there was a remaining line related to a merge conflict
* the docker-run command was improvable (IMHO)
* docker-build is useless if correctly described in the README
* other pep8 remaining

Needs to be tested though.

### CODE REVIEW

File: DOCKER.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ For the following commands sudo might be needed depending on whether your user i
 
 To build the image :
 
-    ./docker-build
+    docker build -t fitiotlab/iot-lab-gateway .
```
Comment (rienafairefr) [inline DOCKER.md]: I'm OK with this, I think we'd better remove the fitiotlab prefix. Even if we might need it if we're pushing the built image to https://hub.docker.com/u/fitiotlab/ (an account we created during the sprint code), it's not needed to build and run locally

**Review by rienafairefr — COMMENTED** [review]
OK, send you a PR with minor stuff in 2s

File: DOCKER.md (hunk 1 above)
Comment (fambon) [reply DOCKER.md]: FYI, probable typo L64 in DOCKER.md    `./docker-run -v` => `docker-run -d`

Comment (rienafairefr) [reply DOCKER.md]: I agree. pushed in PR#1 on your fork

**Comment by aabadie** [comment]
Updated with your pr @rienafairefr.

Might me good if you could confirm that docker-run still works before merging.

**Comment by rienafairefr** [comment]
OK, you've rebased my changes, it's been confusing, but I've seen other bugs when testing docker-run. Right now the api on the dockerized gateway is working. I'll test with an open node soon

**Comment by aabadie** [comment]
Will rebase and squash on of the commits from https://github.com/aabadie/iot-lab-gateway/pull/2

> Right now the api on the dockerized gateway is working. I'll test with an open node soon

Great

**Comment by aabadie** [comment]
Rebase done (just to keep a clean history)

**Comment by rienafairefr** [comment]
OK, just tested on m3, all working. Let's merge it I guess :+1:

**Comment by aabadie** [comment]
Thanks for the reviews and testing @rienafairefr and @fambon

**Pull request merged**